xpeapp_android/app/src/main/java/com/xpeho/xpeapp/ui/components/layout/SidebarItem.kt

```diff
@@ -15,15 +15,17 @@ import androidx.navigation.NavController
 
 @Composable
 fun SidebarItem(
-    navigationController: NavController,
+    navigationController: NavController? = null,
     icon: Painter,
     label: String,
-    route: String
+    route: String,
+    onClick: (() -> Unit)? = null
 ) {
     Row(
         modifier = Modifier
             .clickable {
-                navigationController.navigate(route = route)
+                navigationController?.navigate(route = route)
+                onClick?.invoke()
```

Comment:
`navigationController?.navigate(route = route)` could be inside the `onClick` callback

This way only one parameter is added